portal/src/components/Layout/Layout.tsx

```diff
@@ -37,7 +37,6 @@ export const Layout: React.FC<Props> = ({ children, title, location, isFrontpage
                 <title>{PageTitle}</title>
             </Helmet>
             <ThemeBG />
-            <Header className="jkl-portal__header" />
```

Comment:
Tok vekk menyen i portalen for å unngå at den flashet over skjermen og ble med i skjermbildet.

Mulig vi må sette portalen i testmodus for å unngå noen ting som ikke er nødvendig for automatiske tester.

Reply:
Dette er også et resultat av samme bug som i Accordion, som nå er fikset i master 😊 Men er godt mulig det er andre ting som uansett blir lettere ved å ha en testmodus.